customskill/response/types.go

```diff
@@ -0,0 +1,38 @@
+package response
+
+type Envelope struct {
+	Version           string                 `json:"version"`
+	SessionAttributes map[string]interface{} `json:"sessionAttributes,omitempty"`
+	Response          Response               `json:"response"`
+}
+
+type Response struct {
+	OutputSpeech     *OutputSpeech `json:"outputSpeech,omitempty"`
+	Reprompt         *Reprompt     `json:"reprompt,omitempty"`
+	Card             *Card         `json:"card,omitempty"`
+	Directives       []interface{} `json:"directives,omitempty"`
+	ShouldEndSession bool          `json:"shouldEndSession"`
```

Comment:
so I was doing some testing today with the AudioPlayer and VideoPlayer and I think there are some cases where `shouldEndSession` actually needs to be omitted. I was getting an error saying "shouldEndSession flag is not allowed with LaunchVideoApp.LaunchDirective". I"m not sure what the best way to omit this would be. I'm not sure how to do omit empty with a bool type. Maybe it'll require some kind of wrapper type

Reply:
We could omit it using a boolean pointer and `omitempty` in the JSON tag. I will update